src/components/Home/components/Activities.tsx

```
@@ -0,0 +1,128 @@
+          WebkitOverflowScrolling: 'touch',
+        }}
+      >
+        {activity?.list?.map((activity, index) => (
+          <div
+            className={cn(
+              'glass-card p-6 rounded-xl transition-all duration-500 hover:scale-105 hover:shadow-xl',
+              'animate-on-scroll translate-y-4 flex-shrink-0',
+              'w-[280px] md:w-[320px] backdrop-blur-sm',
+            )}
+            style={{ animationDelay: `${index * 150}ms` }}
+          >
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)